llvm/utils/update_cc_test_checks.py

```diff
@@ -309,11 +309,11 @@ def main():
         run_list = []
         line2func_list = collections.defaultdict(list)
 
-        subs = {
-            "%s": ti.path,
-            "%t": tempfile.NamedTemporaryFile().name,
-            "%S": os.path.dirname(ti.path),
-        }
+        subs = [
+            ("%s", ti.path),
+            ("%t", tempfile.NamedTemporaryFile().name),
+            ("%S", os.path.dirname(ti.path)),
+        ] + ti.lit_substitutions
```

Comment:
Nit: this could also be written as `[..., *ti.lit_substitutions]`.

Reply:
Oh, TIL